frontend-nannyway/src/components/accordion/ChildInCareAccordion.js

```diff
@@ -182,6 +211,49 @@ export default function ChildInCareAccordion({kid}) {
                                     </Typography>
                                 </ListItem>
                             </List>
+                            <Divider/>
+                            <Grid
+                                container
+                                direction="row"
+                                justify="center"
+                                alignItems="center"
+                            >
+                                <Box m={1}>
+                                    <Button
+                                        variant="contained"
+                                        className={classes.deleteButton}
+                                        onClick={handleClickOpen}
+                                        startIcon={<DeleteForeverIcon/>}
+                                    >
+                                        {kid.firstName} löschen
+                                    </Button>
+                                </Box>
+                                <Dialog
```

Comment:
you could move the dialog to a separate component like "ConfirmDialog"

Reply:
Ran into some issues regarding the useState. Will leave the code as it is for now, but opened an according todo for the next days ;)